src/components/admin.tsx

```diff
@@ -789,12 +792,18 @@ function Admin({ user, isConductor = false }: adminProps) {
   };
 
   useEffect(() => {
-    checkTraceLangStatus(`${code}`).then((snapshot) =>
-      setTrackLanguages(snapshot.val())
-    );
-    checkTraceRaceStatus(`${code}`).then((snapshot) =>
-      setTrackRace(snapshot.val())
-    );
+    checkTraceLangStatus(`${code}`).then((snapshot) => {
+      setTrackLanguages(snapshot.val());
+      if (trackLanguages) {
+        setPolicy(new LanguagePolicy(2, "e"));
```

Comment:
Same point about code maintainability here.

Reply:
The 2 will eventually have to be configurable elsewhere, because it is 4 for some other users. In the interim, it is immediately useful here, but it will definitely be moved away in the near future.